docker/examples/elastic/pygeoapi/docker.config.yml

```diff
@@ -86,41 +86,6 @@ metadata:
         role: pointOfContact
 
 resources:
-    obs:
```

Comment:
@krishnaglodha Thank you for this, but this is actually addressing a different issue https://github.com/geopython/pygeoapi/issues/1362 . Please don't address it in this PR.

Reply:
@krishnaglodha Please don't do anything else and read the other comments from @PascalLike and myself.

Reply:
Yes will not work on it